source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,267 @@
+
+1. From your local terminal, use terminus to create a site on Pantheon:
+
+    ```nohighlight
+    terminus site:create $SITE "My Site" "Drupal 8" --org="My Team"
+    terminus connection:set $SITE.dev git
+    ```
+
+2. Add the Pantheon remote repository address and push the code to it:
+
+    ```nohighlight
+    PANTHEON_REPO=$(terminus connection:info $SITE.dev --field=git_url)
+    git remote add pantheon $PANTHEON_REPO
+    git push --force pantheon master
+    ```
+
+3. Complete the Drupal site configuration on Pantheon, replacing the values for `--site-mail`, `--account-mail`, and `--account-name`:
+
+    ```nohighlight
+    terminus build-env:site-install --site-mail="<your email>" --site-name="My Drupal Site" --account-mail="<your email>" --account-name="admin" $SITE.dev
+    ```
+
+4. Verify the site is installed and working:
+
+    ```nohighlight
+    terminus env:view $SITE.dev
+    ```
+
+    Now the master branch of GitHub, your local, and Pantheon are in sync.
+
+
+
+## Configure Jenkins
+
+### Create the Jenkins Project
+
+1. Log into the Jenkins dashboard as an admin user. Click on **New Item**.
+
+    ![Jenkins dashboard](/source/docs/assets/images/integrations/jenkins/jenkins_dash.png)
+
+2. Give the project a name with no spaces. Select **Freestyle Project** and click **OK** to save.
+
+    ![New Jenkins project](/source/docs/assets/images/integrations/jenkins/new_job.png)
+
+3. The next page lets you configure options for this project. In the **General** tab, select "GitHub project" and enter the repository URL (e.g. `https://github.com/YOUR-ORG/YOUR-PROJECT`).
+
+4. In the **Source Code Management** tab select **Git**, then **Advanced**. Add the following information:
+
+    - **Repository URL**: The path to the .git file of your repository, e.g. `https://github.com/YOUR-ORG/YOUR-PROJECT.git`
+    - **Name**: `origin`
+    - **Refspec**: `+refs/heads/*:refs/remotes/origin/*`
+    - **Branch specifier**: `origin/*`
+    - **Additional Behaviours**: Add **Prune stale remote-tracking branches**, and **Clean after checkout**:
+
+    ![SCM view](/source/docs/assets/images/integrations/jenkins/scm_settings.png)
+
+5. We want code changes to trigger our build (as opposed to setting up a periodic build, for example). Under **Build Triggers**, Check the box labelled, "GitHub hook trigger for GITScm polling".
+
+6. Under **Build Environment**, check the box labelled "Inject environment variables to the build process."
+
+    The first new field to appear is the **Properties File Path**. The convention is to add these files to `/var/lib/jenkins/`. Verify the file path exists on the Jenkins server, or create a file and add the path accordingly. In our example we create a file called `envVars.properties` in the standard path.
```

Comment:
`Verify the file path exists on the Jenkins server, or create a file and add the path accordingly.` Sounds like there may already be a value here, or a standard location on the server. Are either of these suppositions true?